shared-ui/src/main/res/layout/layout_progess_sheet.xml

```diff
@@ -1,75 +1,71 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    xmlns:tools="http://schemas.android.com/tools"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:padding="24dp"
     android:background="@drawable/drag_anchor_drawable"
-    >
+    android:padding="20dp">
+
     <TextView
         android:id="@+id/standard_sheet_title"
+        style="@style/Tella_Main_White_Text.Bold"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        tools:text="Download incident"
-        style="@style/Tella_Main_White_Text.Bold"
-        android:textSize="16sp"
         android:layout_marginStart="50dp"
-        app:layout_constraintTop_toTopOf="parent"
+        android:textSize="16sp"
         app:layout_constraintStart_toStartOf="parent"
-        />
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Download incident" />
 
     <com.google.android.material.progressindicator.CircularProgressIndicator
         android:id="@+id/progress_circular"
-        android:layout_width="20dp"
-        android:layout_height="20dp"
-        app:layout_constraintTop_toBottomOf="@id/standard_sheet_title"
-        app:layout_constraintStart_toStartOf="parent"
+        style="@style/Tella_Progress_Circular"
+        android:layout_marginTop="0dp"
+        android:indeterminate="false"
         android:progressTint="@color/tigers_eye"
-        android:layout_marginStart="20dp"
-        android:layout_marginTop="10dp"
-        app:trackColor="@color/wa_white"
-        app:indicatorColor="@color/tigers_eye"
-        app:trackThickness="4dp"
-        />
+        android:visibility="visible"
+        app:indicatorSize="20dp"
+        app:layout_constraintBottom_toBottomOf="@+id/tv_progress_indicator"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@id/tv_progress_indicator"
+        app:trackThickness="3dp" />
 
     <TextView
         android:id="@+id/tv_progress_indicator"
+        style="@style/Tella_Main_White_Text.Meduim"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        tools:text="90% complete"
-        style="@style/Tella_Main_White_Text.Meduim"
+        android:layout_marginStart="50dp"
```

Comment:
We have a standard margin in the project , please check it `activity_horizontal_larger_margin`